Samples/PushNotifications/cpp-console/cpp-console.cpp

```diff
@@ -0,0 +1,199 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <windows.h>
+#include <iostream>
+
+//#include <unknwn.h>
+#include <appmodel.h>
+#include <wil/result.h>
+#include <wil/cppwinrt.h>
+//#include <wil/resource.h>
+
+#include <winrt/Windows.ApplicationModel.Activation.h>
+#include <winrt/Windows.ApplicationModel.Background.h>
+#include <winrt/Windows.Foundation.h>
+//#include <winrt/Windows.Foundation.Collections.h>
+//#include <winrt/Windows.Storage.h>
+//#include <winrt/Windows.Storage.Streams.h>
+#include <winrt/Microsoft.Windows.AppLifecycle.h>
+#include <winrt/Microsoft.Windows.PushNotifications.h>
+#include <winrt/Windows.Globalization.DateTimeFormatting.h>
+
+#include "winrt\Windows.Foundation.h"
+//#include "winrt\Windows.Foundation.Collections.h"
+//#include "winrt\Windows.ApplicationModel.Resources.h"
+
+//using namespace winrt;
+//using namespace winrt::Windows::ApplicationModel::Resources;
+
+using namespace winrt::Microsoft::Windows::AppLifecycle;
+using namespace winrt::Microsoft::Windows::PushNotifications;
+using namespace winrt::Windows::ApplicationModel::Activation;
+using namespace winrt::Windows::ApplicationModel::Background; // BackgroundTask APIs
+using namespace winrt::Windows::Foundation;
+//using namespace winrt::Windows::Storage;
+//using namespace winrt::Windows::Storage::Streams;
+using namespace winrt::Windows::Globalization::DateTimeFormatting;
+
+// To obtain an AAD RemoteIdentifier for your app,
+// follow the instructions on https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app
+winrt::guid remoteId{ "00000000-0000-0000-0000-000000000000"}; // Replace this with own remoteId
```

Comment:
Please address the comments from the unpkg sample PR applicable to this PR.

Reply:
All feedback made into the unpkg sample PR up to now, have equaly been addressed in this PR.